src/atoms/Icon/icons.module.scss

```diff
@@ -0,0 +1,19 @@
+@import 'src/assets/stylesheets/global';
+
+.icon-wrapper {
+  width: 100%;
```

Comment:
@drewdrewthis gotcha - ok remove the duplicate styles & fix the examples and then merge